package.json

```diff
@@ -3,7 +3,13 @@
   "version": "2.7.2",
   "description": "Cosa Models for MongoDB",
   "main": "lib/index.js",
+  "engines": {
+    "node": "8.9.0",
```

Comment:
I could see you wanting to drop this down- not sure what requirements you might have in your other projects. Everything on the product side is up at `8.9.0`.